{{cookiecutter.repo_name}}/src/{{cookiecutter.repo_name_slug}}/{{cookiecutter.conversion_name}}/{{cookiecutter.conversion_name}}_requirements.txt

```diff
@@ -1,3 +0,0 @@
-nwb-conversion-tools==0.11.1   # Example of specific pinned dependecy
-some-extra-package==1.11.3  # Example of another extra package that's necessary for the current conversion
```

Comment:
Did you feel that the other examples were too much detail? 

```
some-extra-package==1.11.3  # Example of another extra package that's necessary for the current conversion
roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@8db5f9cb3a7ee5efee49b7fd0b694c7a8105519a # Github pinned dependenc
```

I like the last one as a reminder to pin to specific commit of github for roiextractor or spikeinterface. When doing past projects they were changing quickly but maybe this is not a good practice anymore.